deployment/secure_research_desktop/packages/packages-python.yaml

```diff
@@ -69,15 +61,9 @@ packages:
     statsmodels:
         "all": [">0.10.0"]  # increase solver flexibility
     sympy:
-    tables:
     tensorflow:
         "3.11": ["uninstallable"]  # does not yet support 3.11
     thinc:
-    torch:
-    torchvision:
```

Comment:
Instead of removing `torchvision` could you mark it as `uninstallable` for all Python versions and explain why?